### DISCUSSION TITLE
Change quicktool size and control LED

### DISCUSSION THREAD

**Discussion by thienntm** [Valkirie/HandheldCompanion#758] [Q&A] [answered]
Hi, I'm using HC to replace Ayaspace2 and I have some questions after a few days using it.
1. Are there any functions to resize quicktool or make it flexible base on screen resolution? My ayaneo air plus set scale at 175%.
Sometimes I changed resolution to 720p in quicktool to match game config, but quicktool still in original size (it's too big in 720p) and missing a part.
2. How long till the next update for LEDs control? Using in the dark is terrible cause of bright joystick LEDs.

**Comment by CasperH2O** [comment, marked as answer]
Hey @thienntm .

1. This used to be possible in older versions, but it should now autoscale properly for any resolution and scale, so it would seem you have found a bug.
2. LED control is currently still on the list to do, there is no expected date. If you are skilled in C# we do have a design and information on how to implement this. See also: #354
